eureka2-client/src/main/java/com/netflix/eureka2/client/transport/ResolverBasedTransportClient.java

```diff
@@ -76,7 +79,7 @@ public MessageConnection call(
                                             ObservableConnection<Object, Object> conn) {
                                         return new SelfClosingConnection(
                                                 new HeartBeatConnection(
-                                                        new BaseMessageConnection("client", conn, metrics),
+                                                        new BaseMessageConnection(clientId, conn, metrics),
```

Comment:
By id I meant as an append to the clientName, since we can have multiple generations of MessageConnection per client.